README.md

```diff
@@ -1 +1,523 @@
+│                                               │                           │
+│             Human Configuration:              │                           │
+│  Defaults, Exception Handling, Comments, Tags │                           │
+│               (IPVM Workflow)                 │                           │
+│                                               │  Multi-Request Pipelining │
+├───────────────────────────────────────────────┤     (UCAN Invocation)     │
+│                                               │                           │
+│      IPVM Config, Verification Level, etc     │                           │
+│                 (IPVM Task)                   │                           │
+│                                               │                           │
+├───────────────────────────────────────────────┴───────────────────────────┤
+│                                                                           │
+│                                 Call Graph                                │
+│                             (UCAN Invocation)                             │
+│                                                                           │
+├───────────────────────────────────────────────────────────────────────────┤
+│                                                                           │
+│                                 Authority                                 │
+│                                (UCAN Core)                                │
+│                                                                           │
+└───────────────────────────────────────────────────────────────────────────┘
+```
+
+# 2 Envelope
+
+The outer wrapper of a workflow MUST contain the following fields:
+
+| Field           | Type       | Description                                                             | Required |
+|-----------------|------------|-------------------------------------------------------------------------|----------|
+| `ipvm/workflow` | `Workflow` | IPVM Workflow                                                           | Yes      |
+| `signature`     | `VarSig`   | [VarSig](https://github.com/ChainAgnostic/varsig/) of serialized fields | Yes      |
+
+| Field      | Type                         | Description                                                            | Required | Default |
+|------------|------------------------------|------------------------------------------------------------------------|----------|---------|
+| `v`        | `"0.1.0"`                    | IPVM workflow version                                                  | Yes      |         |
+| `meta`     | `{String : Any}`             | User-defined object (tags, comments, etc)                              | No       | `{}`    |
+| `parent`   | `[&Workflow, Label] or Null` | The workflow & task label that initiated the current workflow (if any) | No       | `Null`  |
```

Comment:
Minus general provenance/lineage, @expede how do you see `parent` being used in an implementation of workflow(s)?

Reply:
Entirely for provenance & tracing